coretex/cli/modules/node.py

```diff
@@ -236,6 +236,30 @@ def selectNodeMode(storagePath: str) -> Tuple[int, Optional[int]]:
     return availableNodeModes[selectedMode], None
 
 
+def promptCpu(config: Dict[str, Any], cpuLimit: int) -> int:
+    cpuCount: int = clickPrompt(f"Enter the number of CPUs the container will use (Maximum: {cpuLimit}) (press enter to use default)", cpuLimit, type = int)
+
+    if cpuCount > cpuLimit:
+        errorEcho(f"ERROR: CPU limit in Docker Desktop ({cpuLimit}) is lower than the specified value ({cpuCount})")
+        return promptCpu(config, cpuLimit)
+
+    return cpuCount
+
+
+def promptRam(config: Dict[str, Any], ramLimit: int) -> int:
+    nodeRam: int = clickPrompt(f"Node RAM memory limit in GB (Minimum: 6GB, Maximum: {ramLimit}) (press enter to use default)", ramLimit, type = int)
```

Comment:
Why is minimum value not extracted from a constant? Also minimum has GB notation while maximum will just display a number without any notation